### PR TITLE
Sc 16784 Added TagDto

### PR DESCRIPTION
**Pull request by KoNekoD**

### CODE REVIEW

**Review by osipal — APPROVED** [review]
I have no idea what this is for. It’s better to appoint someone as a reviewer who knows why this class was added. The PR description is also empty

P.S. And, if possible, a short version might be preferable
```php
class TagDto
{
  public function __construct(
    public readonly int $id,
    public readonly string $name,
  ) {}
}
```

**Review by nkylinenko — APPROVED** [review]

File: src/TagDto.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+declare(strict_types=1);
+
+namespace Systeme\Component\Dto;
+
+class TagDto
+{
+    private int $id;
+    private string $name;
+
```
Comment (bitpetr) [inline src/TagDto.php:11]: `createdAt`?

**Review by bitpetr — APPROVED** [review]

File: src/TagDto.php (hunk 1 above)
Comment (KoNekoD) [reply src/TagDto.php:11]: Do you think we should add it? It wouldn't have any value.

Comment (bitpetr) [reply src/TagDto.php:11]: We have it in dashboard model.
How did you estimate the value? :grin:

Comment (KoNekoD) [reply src/TagDto.php:14]: done 33ebc57e31cf8c8146557dfd00f8cd66401392e1

**Pull request merged**